wafflehaus/nova/networking/network_count_check.py

```diff
@@ -119,7 +119,7 @@ def _get_networks(body):
         networks = body.get("networks")
         if networks is None:
             return None
-        return [n["uuid"] for n in networks]
+        return [n["uuid"] for n in networks if "uuid" in n]
```

Comment:
Yea the only way to resolve it would be to make the GET /networks call over to neutron, without that we can't really support port attachments, so I would say return a 403 if any element does not have a uuid key.